lib/Interface/core/Container.cpp

```diff
@@ -32,7 +32,7 @@
 // MemRef Shape Constructor.
 // Construct a MemRef object from the data shape and initial value.
 // The default initial value is 0.
-template <typename T, std::size_t N>
+template <typename T, size_t N>
```

Comment:
My mild opinion on this is to prefer `std::size_t` over `size_t` simply because the latter takes care of namespace simplification directly on the user end, making things more simple (and somewhat less problematic) for the compiler. I am not really sure if both of them have a different effect on readability. I am open to further discussion on this.

Reply:
IMO there is no real difference between `std::size_t` and `size_t`, neither in program correctness nor in understandability, so both of them are OK to use. Though using `std::size_t` is somewhat more concrete, I will vote for `size_t` since this shortcut is more frequent to use/type and, particularly in the old version of this file, `size_t` outnumbers the other. So I think to replace `std::size_t` with `size_t` is more suitable.

Reply:
My opinion is to change `size_t` instances to `std::size_t` in this file (for consistency).